pages/api/issue_card.js

```diff
@@ -15,20 +15,53 @@ export default async function handler(req, res) {
     });
 
     const financialAccount = financialAccounts.data[0];
+    const cardholder = await stripe.issuing.cardholders.retrieve(
+      req.body.cardholderid,
+      { stripeAccount: StripeAccountId },
 
-    const card = await stripe.issuing.cards.create(
-      {
-        cardholder: req.body.cardholderid,
-        financial_account: financialAccount.id,
-        currency: 'usd',
-        type: 'virtual',
-        status: 'active',
-      },
-      {stripeAccount: StripeAccountId},
     );
 
+    if (req.body.card_type == 'physical') {
+
+      const card = await stripe.issuing.cards.create(
+        {
+          cardholder: req.body.cardholderid,
+          financial_account: financialAccount.id,
+          currency: 'usd',
+          shipping: {
+            name: cardholder.name,
+            service: 'standard',
+            type: 'individual',
+            address: {
+              line1: cardholder.billing.address.line1, 
```

Comment:
i think we can do some destructuring here or maybe pass address: cardholder.billing.address?

Reply:
Fixed on commit [779b1b0](https://github.com/stripe-samples/treasury-nextjs/pull/19/commits/779b1b01fe53154e3ad6d07714f1c83c51607c17)